app/src/main/java/org/mifos/mobile/ui/fragments/RegistrationFragment.kt

```diff
@@ -223,7 +231,11 @@ class RegistrationFragment : BaseFragment(), RegistrationView {
         super.onDestroyView()
         presenter?.detachView()
     }
+    private fun checkValidity(ccp: CountryCodePicker): Boolean {
```

Comment:
What about if we rename it to `isPhoneNumberValid` and use a full name param instead of `ccp`?